package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@amino-ui/core",
   "version": "0.29.0",
   "description": "Core UI components for Amino",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
This will enable imports to use `@amino-ui/core` instead of `@amino-ui/core/dist`. This should also improve the auto-completion generated by IDEs that do static analysis based on a package's declarations